closure/bin/build/jscompiler.py

```diff
@@ -124,12 +128,16 @@ def Compile(compiler_jar_path, source_paths,
 
   java_version = _ParseJavaVersion(str(_GetJavaVersionString()))
 
+  flags_file = tempfile.NamedTemporaryFile(delete = False)
   args = _GetJsCompilerArgs(
-      compiler_jar_path, java_version, source_paths, jvm_flags, compiler_flags)
+      compiler_jar_path, java_version, source_paths, flags_file, jvm_flags, compiler_flags)
```

Comment:
Passing a `flags_file` handle feels very strange here. I think a better pattern would be something like:

```
args = getjscompilerargs(...)
flag_file = getflagfile(....)
args += ['--flag_file', flag_file.name]
```

With this you have a nice separation of concerns and you're not left guessing what `GetJsCompilerArgs` does with a flags file you pass it.